### PR TITLE
Some fixups for Thunderbolt

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/thunderbolt/fu-thunderbolt-device.c [hunk 1]
```diff
@@ -776,7 +776,8 @@ fu_thunderbolt_device_write_firmware (FuDevice *device,
 	}
 
 	/* whether to wait for a device replug or not */
-	if (!fu_device_has_flag (device, FWUPD_DEVICE_FLAG_USABLE_DURING_UPDATE)) {
+	if (self->device_type != FU_THUNDERBOLT_DEVICE_TYPE_RETIMER &&
```
Comment (djcampello) [inline plugins/thunderbolt/fu-thunderbolt-device.c:779]: I am not entirely sure this is always the case. This might be true only on the NDA case.

**Review by djcampello — APPROVED** [review]

**Comment by djcampello** [comment]
I guess I approved both changes by mistake. See my previous comment

File: plugins/thunderbolt/fu-thunderbolt-device.c (hunk 1 above)
Comment (superm1) [reply plugins/thunderbolt/fu-thunderbolt-device.c:779]: OK I'll take just the other one until we can confirm what happens in device attached case.

**Review by hughsie — APPROVED** [review]
Mea culpa.

**Pull request merged**